### PR TITLE
Create ImageClassificationSVC.ipynb

### PR DESCRIPTION
**Pull request by SaharshLaud**

### CODE REVIEW

**Comment by Thilakraj1998** [comment]
Resize the sample image just for display to 50x50
also same for training reduce image size
image size effects the training period

**Comment by SaharshLaud** [comment]
Reduced the image size

**Comment by Thilakraj1998** [comment]
define a split ratio for train and test.
if you observe you accuracy in last few submission it differ from each other, that due to random_state not specified
so add random_state in both train_test_split and model training

**Comment by SaharshLaud** [comment]
I have added a split ratio as well as random_state parameter in the data split so that the accuracy does not fluctuate.
